### ISSUE TITLE
Task 1 LIDAR rules

### ISSUE THREAD

**Issue by LM239** [Sjyhne/MapAI-Competition#8]
Does the prohibition against LIDAR in task 1 cover all steps of the task, or are we within the rules as long as we do not use lidar during inference for task 1?

**Comment by dvolgyes** [comment]
I am also interested in that.
My interpretation, but would be nice if @Sjyhne could confirm it:
Since any data is allowed for pretraining, Lidar should be also allowed.
But for inference, it should work with RGB-only.
Is that correct, @Sjyhne ?

**Comment by Sjyhne** [comment]
Task one is meant to only use aerial images for the entire task, which includes the training step. Therefore, you should only use aerial images for training the model. This is specified in the Competition Information part of the readme. Therefore, it is not allowed to use Lidar for training. Sorry for not being explicit regarding this manner. I did not recognize the use of Lidar for training for task 1, which is why it is not mentioned.

In the other issue (#3), I stated that additional data was allowed. However, I should probably have stated explicitly that additional data must follow the rules for the task.

**Comment by dvolgyes** [comment]
Could you specify specify the rules for the task part?
No offense, but the description is a bit short:
" 1.  Building segmentation only using aerial images.
  2.  Building segmentation using lidar data (it is allowed to combine the lidar with aerial images). "

E.g., which one of these are allowed or not allowed?
- using ImageNet pretraining (rgb but not aerial)
- using aerial imagery from other competitions, like SpaceNet?
- satellite imagery, e.g. BigEarthNet? remote sensing, but hyperspectral (Sentinel2)
- using lidar or DEM data from other public sources but not from this competition?
- using pretrained model which was trained on a public dataset (which may or may not have Lidar, or other sources)
- using label data from other sources, like openstreetmap?
- using private imagery (paid sources, like Norge i Bilder) for pretraining? 
If yes, shall the participants disclose it?
- using private label data for training? If yes, shall the participants disclose it? 
E.g. Norwegian building/imagery/etc datasets are accessible to some Norwegian authorities and university projects.
Not us, but other companies e.g. distributors of the Norwegian 'flyfotos' and/or 
might have private dataset over Norway which might be used for pretraining/co-training. 
E.g. I could imagine that the KartAI project has extra data, and might have a competing team given
that both Kartverket and UiA are involved both in KartAI and this competition. 

The #3 stated to be able to use external data, but as you see the details get complex.

- Task 1: If only aerial imagery allowed, does this rule out Imagenet, Sentinel-2, Pleiades, Worldview, etc?
(None of them are aerial. Either classic photography or satellite.)
- Task 1: If aerial imagery allowed, does this include all possible aerial sources, 
  from Spacenet through  Norge i Bilder to private sources?

- So is there any public data source which is NOT allowed in Task 2?
E.g are open satellite altimeters (GEDI, IceSat or CryoSat-2) allowed?

@Sjyhne I am sorry for the lot of questions, but I think there might be many corner cases,
and better to discuss it now than after submission.

**Comment by Sjyhne** [comment]
Thank you for your comprehensive feedback! Highly appreciate it, in retrospect, I can see that multiple things have been too vague, which is why your feedback is so valuable. I'll formulate a more concise description for each of the tasks - I will update the readme tomorrow with a better description of the do's and don't's for both tasks. 

Don't apologize! Questions like these are crucial for the improvement of both the competition setup and descriptions.

**Comment by Sjyhne** [comment]
There, I've now updated the readme. https://github.com/Sjyhne/MapAI-Competition/commit/b74c390e7e673cc7cef1631baefe6be476ffb480
